measures/ResStockArguments/measure.rb

```diff
@@ -93,6 +93,11 @@ def arguments(model)
     arg.setDefaultValue(0.6)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('wall_assembly_r_increase', false)
```

Comment:
We don't need this argument.

Reply:
👍 [The issue](https://github.com/NREL/resstock/issues/42) is just about upgrade costing (cost multiplier) for ceiling insulation, not the energy model.